tests/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/test_source_pings/moz-fx-data-shared-prod.telemetry.first_shutdown.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 - client_id: client-3
   sample_id: 30
-  submission_timestamp: '2023-06-01 11:03:00'
+  submission_timestamp: '2023-06-01 10:03:00'
```

Comment:
Changed these so the `new_profile` ping arrives second but is still chosen over the `first_shutdown` ping